### DISCUSSION TITLE
Documentation Site

### DISCUSSION THREAD

**Discussion by appinteractive** [fujidaiti/smooth_sheets#31] [General]
Hey @fujidaiti currently can't provide much functional contributions but I could kickstart a documentation site if you would be interested. 

I'm asking before doing some work on it as I don't know if you already have something in the works.

**Comment by fujidaiti** [comment]
> I could kickstart a documentation site

That sounds interesting! Can you explain it to me in more detail since I'm not quite sure what it's like?

**Reply by appinteractive** [reply]
I could provide you with some hint like that site: 

https://awesome-notification-docs-kjgaagozd-appinteractive.vercel.app/

That's one I prepared for another flutter Projekt.  

The documentation site is build from markdown files and then automatically deployed and searchable (when configured)

**Reply by fujidaiti** [reply]
That would be great! But right now, this package is under construction and is not yet stable, so we will have to revise the documentation frequently and may end up creating inconsistencies. When a stable version is released, and if you are willing and have time to create the documentation site, it would be greatly appreciated :)
